src/pages/Thanks/ThanksPage.vue

```diff
@@ -377,6 +399,21 @@ export default {
 					this.shareAskCopyVersion,
 				);
 			}
+
+			// MARS-310 Category Share on Thanks page
+			const categoryShareResult = this.apollo.readFragment({
+				id: 'Experiment:share_ask_copy',
+				fragment: experimentVersionFragment,
+			}) || {};
+
+			this.categoryShareVersion = categoryShareResult?.version;
+			if (this.categoryShareVersion) {
```

Comment:
I think we also need to check if the user has a loan that qualifies for this experiment before we track them.